chia/util/chia_logging.py

```diff
@@ -29,7 +29,7 @@ def initialize_logging(service_name: str, logging_config: Dict, root_path: Path)
         logger.addHandler(handler)
     else:
         logger = logging.getLogger()
-        handler = ConcurrentRotatingFileHandler(log_path, "a", maxBytes=20 * 1024 * 1024, backupCount=7)
+        handler = ConcurrentRotatingFileHandler(log_path, "a", maxBytes=20 * 1024 * 1024, backupCount=logging_config.get("log_maxfilesrotation", 7))
```

Comment:
This is great and thank you for your PR. We will need one more item - you can't assume that the entry is in config.yaml so please add the default setting we have now if you don't find it in the config so it is fully backwards compatible.

Reply:
![image](https://user-images.githubusercontent.com/6495665/115925584-af141f00-a481-11eb-8f85-f5d94448ca67.png)
It's work if there isn't the key. I try it.
![image](https://user-images.githubusercontent.com/6495665/115925801-09ad7b00-a482-11eb-8d75-04bbe6b51b17.png)
If there isn't the default is 7

Reply:
I use get of dictionary with a second parameter for default. The syntax is
![image](https://user-images.githubusercontent.com/6495665/115926057-76c11080-a482-11eb-9ce9-378095342346.png)

Reply:
I think it's backwards compatible. Are you agree?